test-case/check-pause-release-suspend-resume.sh

```diff
@@ -51,22 +52,14 @@ set -e
 ##  * No unexpected errors should be present in dmesg during or after test
 ##      completion.
 
-source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+CASEDIR=$(dirname "${BASH_SOURCE[0]}")
+source $CASEDIR/../case-lib/lib.sh
```

Comment:
Fix the quoting issues reported by shellcheck. Not all the issues in this script but at least all the ones in new or modified code. Don't add new shellcheck warnings.

Reply:
```shell
# shellcheck source=...
```
See how other scripts do it.